integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -335,6 +335,48 @@ void testAlterTableRenameColumn() {
     checkTableColumns(tableName, renameColumns, getTableInfo(tableName));
   }
 
+  @Test
+  void testUpdateColumnPosition() {
+    String tableName = "test_column_position";
+    dropTableIfExists(tableName);
+
+    List<SparkColumnInfo> simpleTableColumns = getSimpleTableColumn();
+
+    sql(
+        String.format(
+            "CREATE TABLE %s (id INT COMMENT 'id comment', name STRING COMMENT '', age INT) USING PARQUET",
```

Comment:
suggest make column the same type like string,  there may some unexpected errors with different column type when change position

Reply:
okay, i'll try.

Reply:
Hi @FANNG1 could you help review this again? thank you.